pkg/cmd/server/bootstrappolicy/all_test.go

```diff
@@ -0,0 +1,121 @@
+package bootstrappolicy
+
+import (
+	"reflect"
+	"strings"
+	"testing"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/util/diff"
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+const osClusterRoleAggregationPrefix = "system:openshift:"
```

Comment:
Do we really need this constant if we use it only once? 5 lines bellow we don't use it, for example.